crates/bevy_hierarchy/src/child_builder.rs

```diff
@@ -240,42 +246,45 @@ impl Command for RemoveParent {
     }
 }
 
-/// Struct for building children onto an entity
+/// Struct for building children entities and adding them to a parent entity.
 pub struct ChildBuilder<'w, 's, 'a> {
     commands: &'a mut Commands<'w, 's>,
     push_children: PushChildren,
 }
 
 impl<'w, 's, 'a> ChildBuilder<'w, 's, 'a> {
-    /// Spawns an entity with the given bundle and inserts it into the children defined by the [`ChildBuilder`]
+    /// Spawns an entity with the given bundle and inserts it into the parent entity's [`Children`],
+    /// defined by the [`ChildBuilder`], later adding a [`Parent`] component to the created entity.
     pub fn spawn(&mut self, bundle: impl Bundle) -> EntityCommands<'w, 's, '_> {
         let e = self.commands.spawn(bundle);
         self.push_children.children.push(e.id());
         e
     }
 
-    /// Spawns an [`Entity`] with no components and inserts it into the children defined by the [`ChildBuilder`] which adds the [`Parent`] component to it.
+    /// Spawns an [`Entity`] with no components and inserts it into the parent entity's [`Children`],
+    /// defined by the [`ChildBuilder`], later which adds the [`Parent`] component to the created
+    /// entity.
     pub fn spawn_empty(&mut self) -> EntityCommands<'w, 's, '_> {
         let e = self.commands.spawn_empty();
         self.push_children.children.push(e.id());
         e
     }
 
-    /// Returns the parent entity of this [`ChildBuilder`]
+    /// Returns the parent entity of this [`ChildBuilder`].
     pub fn parent_entity(&self) -> Entity {
         self.push_children.parent
     }
 
-    /// Adds a command to this [`ChildBuilder`]
+    /// Adds a command to [`Commands`] of a parent of this [`ChildBuilder`].
```

Comment:
I think this should remove references to `ChildBuilder` since it's irrelevant to what the added command does. Here is an imperfect suggestion:

```suggestion
    /// Adds a command, like [`Commands::add`].
```